### DISCUSSION TITLE
Does this project support llama-2 model?

### DISCUSSION THREAD

**Discussion by MaratZakirov** [ggml-org/llama.cpp#2328] [Q&A] [answered]
Recently https://github.com/facebookresearch/llama have some instructions to download llama-2 so my question is does this project support llama-2? Or where can I download original llama model weights?

**Comment by izard** [comment, marked as answer]
Smaller models just work, biggest one almost works. Try hugginface/Thebloke, or go the official way from Facebook, then convert to ggml.

**Comment by ianscrivener** [comment]
Yes, llama.cpp supports Llama2, Llama, Vicuna, Alpaca, and many more...

[TheBloke has LOTS of models on HuggingFace](https://huggingface.co/models?search=thebloke%20ggml): try a **7B model**, make sure it is **ggmlv3** and check in the Model Card that it works on llama.cpp
